index.html

```diff
@@ -0,0 +1 @@
+Hello there, awesome person!
```

Comment:
```suggestion
Hello there, awesome person!
```

Reply:
<html>
    <head>
        <title>My web page</title>
    
    </head>

  <body>
<">

 <ol>
    <li>Signal Officer in the Army</li>
    <li>Mother of 2 Dogs</li>
      <li>My favorite hobby is painting</li>
</ol>
      
  </body>

</html>

Reply:
```suggestion
Hello there, awesome person!
```
<html>
   
 <body>
      
```suggestion
Hello there, awesome person!
```

 </body>

</html>